packages/suite/src/types/suite/guide.tsx

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable camelcase */
+export type GuideView = 'GUIDE_DEFAULT' | 'GUIDE_CATEGORY' | 'GUIDE_ARTICLE';
+
+export type FeedbackView = 'FEEDBACK_TYPE_SELECTION' | 'FEEDBACK_BUG' | 'FEEDBACK_SUGGESTION';
+
+export type ActiveView = GuideView | FeedbackView;
+
+export type Category =
+    | 'dashboard'
+    | 'account'
+    | 'settings'
+    | 'send'
+    | 'receive'
+    | 'trade'
+    | 'other';
+export type Rating = 1 | 2 | 3 | 4 | 5;
+
+export type FeedbackType = 'BUG' | 'SUGGESTION';
+
+export interface UserDataCommon {
+    platform: 'web' | 'desktop';
+    os: string;
+    user_agent: string;
```

Comment:
nit:
You can also track browser, browser version, os and os version if you want. It is ready to use in `env.ts` file. I know that you can get it from userAgent, but just in case...

Reply:
Let's add it in the next version as well

Reply:
@tomasklim the set of fields was chosen as somewhat minimal viable variant we can start with. We definitely plan to iterate on these in future.